### ISSUE TITLE
Add city information to Line/Itinieray names

### ISSUE THREAD

**Issue by AltNico** [grote/osm2gtfs#109]
When [generating national GTFS feeds](https://github.com/grote/osm2gtfs/milestone/1) it is important to have the possibility to add some kind of city information to a Line or Itinerary because it is very likely that one name exists in various cities.

Example: *MGA-110* (Bus number 110 in Managua, Nicaragua)

**Comment by nlehuby** [comment]
In my opinion, your lines should belong to different agencies, and it would make more sense to use agencies rather than cities to differentiate them

**Comment by prhod** [comment]
I agree with @nlehuby . In your exemple, there is an aggregation of different types of information : 
* "Bus" is provided by the route_type
* "110" is provided by the route_short_name 
* "Managua" should either be provided by the agency_name or be a more specific information in the route_name about the line
I quite don't like the idea of transforming the OSM informations in osm2gtfs, unless done in a specific creator.

**Comment by ialokim** [comment]
The only point I see we should definitly change is the use of the route number (aka `ref` tag from OSM) as the `route_id` in the GTFS. It's very likely to encounter more than one routes with the same number that we would like to add to the same GTFS file. One very simple solution would be to use the `master_route` OSM id as GTFS `route_id` (similar to the stops/stations/shapes).

The only downside I see is that it would difficult human readability of the GTFS `routes.txt` and `trips.txt`, but I'm not sure whether this is very important.
